kuma/search/templates/search/base.html

```diff
@@ -4,9 +4,6 @@
 {% if not styles %}
   {% set styles = ('search',) %}
 {% endif %}
-{% if not scripts %}
-  {% set scripts = ('search',) %}
-{% endif %}
```

Comment:
Technically, this would allow each child of `search/base.html` to declare a custom set of `scripts` for itself. But if a child template did not declare its own `scripts`, this `base` template would include the `search` script.